doc/wg/core/notes/core-notes-2023-01-20.md

```diff
@@ -0,0 +1,100 @@
+# Tock Core Notes 2023-01-20
+
+Attendees:
+ - Branden Ghena
+ - Phil Levis
+ - Leon Schuermann
+ - Hudson Ayers
+ - Jett Rink
+ - Alyssa Haroldsen
+ - Vadim Sukhomlinov
+ - Johnathan Van Why
```

Comment:
I signed in at the beginning of the meeting, but had to leave quickly. Probably more accurate to not list me as an attendee than to list me as an attendee (i.e. I did not hear Hudson mention my name during updates).